app/src/main/java/com/codesoom/assignment/common/aop/TaskExceptionHandler.java

```diff
@@ -0,0 +1,27 @@
+package com.codesoom.assignment.common.aop;
+
+import com.codesoom.assignment.common.exceptions.TaskNotFoundException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+/**
+ * 할 일 관련 예외가 던져졌을 떄, 해당 예외에 맞는 Response를 반환합니다.
+ */
+@RestControllerAdvice
+public class TaskExceptionHandler {
```

Comment:
JavaDoc 설명이 뭔가 중언부언하거나 자신감이 없어 보일 땐 이 녀석의 책임이 무엇인지를 짧게 표현하려 해보세요.
```suggestion
/**
 * 할 일과 관련된 예외의 처리를 담당합니다.
 */
@RestControllerAdvice
public class TaskExceptionHandler {
```

Reply:
해당 클래스의 책임은 "예외처리"입니다.
책임에 대해 명확히 하니 클래스에 대한 설명도 간략히 설명이 되는군요!
좋은 팁 감사합니다!